source/common/stream_info/utility.cc

```diff
@@ -116,6 +116,17 @@ absl::optional<std::chrono::nanoseconds> TimingUtility::lastDownstreamRxByteRece
   return duration(timing.value().get().lastDownstreamRxByteReceived(), stream_info_);
 }
 
+absl::optional<std::chrono::duration<double, std::milli>>
+TimingUtility::recordExtAuthzDuration(absl::optional<MonotonicTime> start_time) {
```

Comment:
As @dio said, why is this method in `TimingUtility`?
It should either be a static function called something similar to `timeSince(absl::optional<MonotonicTime> start_time)`, or it can be defined inside the filter code in an empty namespace.

Reply:
I moved it into an empty namespace.